packages/react-ui/src/Fields/Input.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { box, colors, fonts, spacings } from "../theme";
+
+export const Input = ({ name, icon: Icon, ...props }) => {
+  return (
+    <StyledWrapper>
+      <StyledInput name={name} hasIcon={!!Icon} {...props} />
```

Comment:
I prefer the `Boolean(Icon)`syntax, less bizarre

Reply:
OK